RPGManager_development/src/rpg_database/character_sheet/Background.java

```diff
@@ -92,7 +126,11 @@ public void setSelfInSheet(CharacterSheet characterSheet) {
 			throw new InvalidCharacterClassException(String.format("%s is not a %s background!", this.toString(), characterSheet.getData(
 					Fields.BASECLASS).toString()));
 		}
+		characterSheet.characterData.remove(Fields.LANGUAGES);
 		characterSheet.characterData.put(Fields.BACKGROUND, this);
+		LanguagesSet thisLanguages = new LanguagesSet();

```

Comment:
ezt megnézted, vagy csak szerinted van így? A copyconstruktor nem függ attól hogy generikus az osztály vagy sem.
Amúgy a `characterSheet.characterData.remove(Fields.LANGUAGES);`-ot kiszedheted. A put felülírja az előző értéket. (Bocs erre sajnos én is csak ma jöttem rá, de egy sorral kevesebb :) )